glotaran/io/io.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+import xarray as xr
+
+from glotaran.model import Model
+from glotaran.parameter import ParameterGroup
+from glotaran.project import Result
+from glotaran.project import SavingOptions
+from glotaran.project import Scheme
+
+
+class Io:
```

Comment:
`IoPlugin` or `IoInterface` would be a more fitting name.

Reply:
Names are hard.

Reply:
Yeah one of the biggest problems in programming... naming stuff 🤣